cmd/apiserver/main.go

```diff
@@ -165,37 +164,48 @@ func realMain(ctx context.Context) error {
 
 	r.Handle("/health", controller.HandleHealthz(ctx, &cfg.Database, h)).Methods("GET")
 
+	// Make verify chaff tracker.
+	verifyChaffTracker, err := chaff.NewTracker(chaff.NewJSONResponder(encodeVerifyResponse), chaff.DefaultCapacity)
+	if err != nil {
+		return fmt.Errorf("error creating verify chaffer: %v", err)
+	}
+	defer verifyChaffTracker.Close()
+
+	// Make cert chaff tracker.
+	certChaffTracker, err := chaff.NewTracker(chaff.NewJSONResponder(encodeCertificateResponse), chaff.DefaultCapacity)
+	if err != nil {
+		return fmt.Errorf("error creating cert chaffer: %v", err)
+	}
+	defer certChaffTracker.Close()
+
 	{
-		sub := r.PathPrefix("/api").Subrouter()
+		sub := r.PathPrefix("/api/verify").Subrouter()
+		sub.Use(processChaff(verifyChaffTracker))
```

Comment:
process chaff needs to be after requireAPI key for #1222

Reply:
Done